### ISSUE TITLE
Packaging: Cannot run Grafana 9.1.7+ on Alpine 3.15 + LXD

### ISSUE THREAD

**Issue by young-druid** [grafana/grafana#58868]
**What happened**:
Upgraded grafana from 9.1.6 to 9.1.7 (or any other later version)
**/bin/grafana-server -v** gives the following error
**Error relocating /opt/grafana/bin/grafana-server: fcntl64: symbol not found**

**What you expected to happen**:
**/bin/grafana-server -v** works

**Environment**:
- Grafana version: grafana-9.1.7.linux-arm64.tar.gz
- OS Grafana is installed on: Alpine 3.15 arm64 running on LXD 5.8

**Comment by zuchka** [comment]
@young-druid are you building Grafana in an alpine-based container? Can you provide more details and reproduction steps?

**Comment by young-druid** [comment]
@zuchka Not really. I am not trying to build Grafana. I am trying to start grafana server using pre-built binaries, specifically `grafana-9.1.7.linux-arm64.tar.gz`
I am using lxd containers (version 5.8). I created an alpine instance:
`lxc launch images:alpine/3.15 grafana`
So far grafana worked fine for me when I tried to start it inside a container. But starting from version 9.1.7 it stopped working, so I rolled back to 9.1.6.

**Comment by tleopold** [comment]
@zuchka this seems like it could be a backend issue, not a RelEng one.

**Comment by zuchka** [comment]
@young-druid are you still hitting this?

**Comment by young-druid** [comment]
I tried 9.4.3 and I saw the same error. Maybe my setup is not standard, not sure. But currently I am using 9.1.6 and it works.

**Comment by gelicia** [comment]
@young-druid This got shuffled around a bit and lost. Does this still happen with more current versions of Grafana?
